src/coreclr/nativeaot/System.Private.Reflection.Core/src/System/Reflection/Runtime/MethodInfos/RuntimeNamedMethodInfo.cs

```diff
@@ -305,7 +305,36 @@ private RuntimeTypeInfo[] GenericTypeParameters
 
         internal sealed override MethodInvoker GetUncachedMethodInvoker(RuntimeTypeInfo[] methodArguments, MemberInfo exceptionPertainant)
         {
-            return _common.GetUncachedMethodInvoker(methodArguments, exceptionPertainant);
+            MethodInvoker invoker = _common.GetUncachedMethodInvoker(methodArguments, exceptionPertainant, out Exception exception);
+            if (invoker == null)
+            {
+                // If we have byref-like types in the signature, the reason we couldn't find an invoker
+                // is that.
+                bool hasByRefLikeParameter = false;
+                foreach (ParameterInfo parameter in GetParametersNoCopy())
+                {
+                    if (Unwrap(parameter.ParameterType).IsByRefLike)
+                    {
+                        hasByRefLikeParameter = true;
```

Comment:
```suggestion
                        hasByRefLikeParameter = true;
                        break;
```

Reply:
Heh, I had it there and deleted it to save a line of code. This is already in the super slow path.